packages/llvm.rb

```diff
@@ -1,276 +1,104 @@
-      system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_libunwind, '-o', filename_libunwind)
-      abort 'Checksum mismatch. :/ Try again.'.lightred unless
-        Digest::SHA256.hexdigest( File.read("./#{filename_libunwind}") ) == sha256sum_libunwind
-      puts "Libunwind archive downloaded".lightgreen
-    end
-
-    system "tar", "xf", "libunwind-#{version}.src.tar.xz", "-C", "projects"
-    puts "Libunwind source code unpacked".lightgreen
-    system "rm -rf projects/libunwind"
-    system "mv -v projects/libunwind-#{version}.src projects/libunwind"
-
-    ############################# Download test-suite (procjects) ####################################
-    #url_testsuite = "https://releases.llvm.org/#{version}/test-suite-#{version}.src.tar.xz"
-    #uri_testsuite = URI.parse url_testsuite
-    #filename_testsuite = File.basename(uri_testsuite.path)
-    #sha256sum_testsuite = '2ba3cde840c1c0ef744c93ae6a921207dbb9ccbb39f93f7ace9252e5203307cf'
-
-    #if File.exist?(filename_testsuite) && Digest::SHA256.hexdigest( File.read("./#{filename_testsuite}") ) == sha256sum_testsuite
-    #  puts "Unpacking test-suite source code".yellow
-    #else
-    #  puts "Downloading test suite".yellow
-    #  system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_testsuite, '-o', filename_testsuite)
-    #  abort 'Checksum mismatch. :/ Try again.'.lightred unless
-    #    Digest::SHA256.hexdigest( File.read("./#{filename_testsuite}") ) == sha256sum_testsuite
-    #  puts "Test-suite archive downloaded".lightgreen
-    #end
-
-    #  system "tar", "xf", "test-suite-#{version}.src.tar.xz", "-C", "projects"
-    #  puts "Test-suite source code unpacked".lightgreen
-    #  system "rm -rf projects/test-suite"
-    #  system "mv -v projects/test-suite-#{version}.src projects/test-suite"
-
     ############################################################
-    puts "Optional packages are ready".lightgreen
+    puts "Building LLVM Targets: #{LLVM_TARGETS_TO_BUILD}".lightgreen
+    puts "Building LLVM Projects: #{LLVM_PROJECTS_TO_BUILD}".lightgreen
     ############################################################
-
-    puts 'Change to GOLD linker.'.orange
-    original_default = `ld_default g`.chomp
+    ############################################################
+    puts "Setting compile to use python3".lightgreen
+    ############################################################
+    system "grep -rl '#!.*python' | xargs sed -i '1s/python$/python3/'"
+    
+    ############################################################
+    puts "Downloading binutils #{BINUTILS_BRANCH} src to enable gold plugin build".lightgreen
+    ############################################################
+    system "git config --global advice.detachedHead false"
+    # As per https://github.com/SVF-tools/SVF/wiki/Install-LLVM-Gold-Plugin-on-Ubuntu
+    system "git clone --depth 1 --branch #{BINUTILS_BRANCH} git://sourceware.org/git/binutils-gdb.git binutils"
```

Comment:
Why isn't this in the `binutils` package?

Reply:
let's see if that works.